lib/nodes/addon/components/driver-harvester/template.hbs

```diff
@@ -95,12 +98,14 @@
               @class="form-control"
               @content={{namespaceContent}}
               @value={{config.vmNamespace}}
+              @placeholder={{t "nodeDriver.harvester.namespace.placeholder"}}
             />
           {{else}}
             {{input
               type="text"
               value=config.vmNamespace
               classNames="form-control"
+              placeholder=(t "nodeDriver.harvester.vmNamespace.placeholder")
```

Comment:
`nodeDriver.harvester.vmNamespace.placeholder` Doesn't exist in en-us

![image](https://user-images.githubusercontent.com/18697775/142178656-f5e09d73-ce76-4a17-8016-f615c5dfad63.png)

Reply:
Done

Reply:
Done